docs/csharp/language-reference/operators/boolean-logical-operators.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Boolean logical operators - C# Reference"
+description: "Learn about C# operators that perform logical negation, conjunction (AND), and inclusive and exclusive disjunction (OR) operations with Boolean operands."
+ms.date: 04/05/2019
+author: pkulikov
+f1_keywords: 
+  - "!_CSharpKeyword"
+  - "&_CSharpKeyword"
+  - "^_CSharpKeyword"
+  - "|_CSharpKeyword"
+  - "&&_CSharpKeyword"
+  - "||_CSharpKeyword"
+helpviewer_keywords: 
+  - "logical operators [C#]"
+  - "short-circuiting operators [C#]"
+  - "logical negation operator [C#]"
+  - "NOT operator [C#]"
+  - "! operator [C#]"
+  - "AND operator [C#]"
+  - "ampersand operator [C#]"
+  - "conjunction operator [C#]"
+  - "& operator [C#]"
+  - "exclusive OR operator [C#]"
+  - "XOR operator [C#]"
+  - "^ operator [C#]"
+  - "OR operator [C#]"
+  - "disjunction operator [C#]"
+  - "| operator [C#]"
+  - "conditional logical AND operator [C#]"
+  - "short-circuiting logical AND operator [C#]"
+  - "&& operator [C#]"
+  - "conditional logical OR operator [C#]"
+  - "short-circuiting logical OR operator [C#]"
+  - "|| operator [C#]"
+---
+# Boolean logical operators (C# Reference)
+
+The following operators perform logical operations with [bool](../keywords/bool.md) operands:
+
+- Unary [`!` (logical negation)](#logical-negation-operator-) operator.
+- Binary [`&` (logical AND)](#logical-and-operator-), [`|` (logical OR)](#logical-or-operator-), and [`^` (logical exclusive OR)](#logical-exclusive-or-operator-) operators. Those operators always evaluate both operands.
+- Binary [`&&` (conditional logical AND)](#conditional-logical-and-operator-) and [`||` (conditional logical OR)](#conditional-logical-or-operator-) operators. Those operators evaluate the second operand only if it's necessary.
+
+The `&` and `|` operators are also defined for `bool?` operands. For more information, see [The bool? type](../../programming-guide/nullable-types/using-nullable-types.md#the-bool-type) section of the [Using nullable types](../../programming-guide/nullable-types/using-nullable-types.md) article.
+
+For the operands of [integral](../keywords/integral-types-table.md) types, the `&`, `|`, and `^` operators perform bitwise logical operations.
```

Comment:
No links to operators reference by design (though individual sections below have links): I'll add the link to the overview of bitwise operators here. To create such an overview is my next PR.

Reply:
I think that's a good plan.

I am watching traffic patterns to determine when readers land in the wrong article, and which article they are trying to find.